### PR TITLE
V1.5.0-beta2

### PR DESCRIPTION
**Pull request by ryanbaumann**
Fix bugs in v1.5.0-beta1:

* Fix color legend CSS bug
* Fix cluster source broken in the last PR
* Upgrade to GL JS v1.4.1 (bugfix from v1.4.0)
* Fix gl-geocoder marker from the upgrade, default to using proximity flag
* Fix satellite layer not displaying bug
    * This looks to be a bug with PBI Desktop? I will investigate with the Power BI team.
    * Downgrading to v1.0.0 of the Mapbox GL JS API fixes the problem.  It starts to occur in v1.1.0.  However, v1.0.0 is a significantly older version prior to many performance and UX improvements in the Mapbox GL JS SDK.
    * Likely root case is the change to the Cache API in v1.1.0 of GL JS.  

Beta version (w/ gl js v1.0.0)
[mapboxGLMap-v1.5.0-beta2.zip](https://github.com/mapbox/mapboxgl-powerbi/files/3719664/mapboxGLMap-v1.5.0-beta2.zip)

cc/ @sansumbrella @cmtoomey

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,10 +28,8 @@
     "@types/geojson": "^1.0.6",
     "@types/mapbox-gl": "^0.51.4",
     "chroma-js": "^1.3.5",
-    "mapbox-gl": "1.4.0",
+    "mapbox-gl": "^1.0.0",
```
Comment (sansumbrella) [inline package.json]: For consistent dependency versions, we should add `package-lock.json` to version control.

A default `npm install` will get 1.4.1, but if you have an existing `package-lock.json` with an older version, you need to delete the package-lock file and node_modules directory first.

**Comment by otravers** [comment]
Bugs that we saw last week with my demo file with beta 1 that I just verified no longer occur with beta 2:

- color legend CSS bug
- cluster source broken in the last PR
- satellite / satellite streets tilesets (direct or via custom tileset)

Good job! This was tested with today's October 2019 PBI Desktop.

File: src/ts/legendControl.ts [hunk 2]
```diff
@@ -79,24 +79,28 @@ module powerbi.extensibility.visual {
...
 
             data.forEach(({colorStop, color}) => {
+                // Create line item
                 const item = d.createElement('div');
+
+                // Create color element and add to item
                 const colorElement = d.createElement('span');
                 colorElement.setAttribute("class", "mapbox-legend-color middle");
-                colorElement.setAttribute("style", "backgroundColor: color;");
+                colorElement.setAttribute("style", "background-color: " + color + ";");
```
Comment (sansumbrella) [inline src/ts/legendControl.ts]: A less error-prone way of writing this is with template literals:
```
`background-color: ${color};`
```

Comment (yuletide) [reply src/ts/legendControl.ts]: +1 for template literals

File: src/ts/mapboxGeocoderControl.ts [hunk 3]
```diff
@@ -41,7 +42,8 @@ module powerbi.extensibility.visual {
             this.geocoder = new MapboxGeocoder({
                 accessToken: this.accessToken,
                 zoom: this.zoom,
-                trackProximity: false // BUG in mapbox-gl-geocoder
+                trackProximity: true,
```
Comment (yuletide) [inline src/ts/mapboxGeocoderControl.ts]: Per docs this defaults to `true`, we could just remove? https://github.com/mapbox/mapbox-gl-geocoder/blob/master/API.md#parameters

**Pull request merged**